src/main/java/org/openstreetmap/atlas/geography/atlas/README.md

```diff
@@ -0,0 +1,133 @@
+```
+final File atlasFile = new File("/path/to/source.atlas");
+final Atlas atlas = new AtlasResourceLoader().load(atlasFile);
+```
+Query the data. Examples:
+
+* Get all edges within boundaries:
+```
+Rectangle rectangle = ...;
+atlas.edgesIntersecting(rectangle).forEach(edge -> ...);
+```
+* Find all the parks with less than 6 shape points:
+```
+Predicate<Area> filter = area -> {
+    return Validators.isOfType(area, LeisureTag.class, LeisureTag.PARK)
+        && area.asPolygon().size() < 6;
+}
+atlas.areas(filter).forEach(area -> ...);
+```
+or
+```
+Predicate<Area> filter = area -> {
+    return "park".equals(area.getTags().get("leisure"))
+        && area.asPolygon().size() < 6;
+}
+atlas.areas(filter).forEach(area -> ...);
+```
+* Find all buildings with a hole:
+```
+ComplexBuildingFinder finder = new ComplexBuildingFinder();
+Iterables.stream(finder.find(atlas))
+    .filter(complexBuilding -> !complexBuilding.getOutline().inners().isEmpty())
+    .forEach(complexBuilding -> ...);
+```
+* How many `Edge`s are connected to a `Node`:
+```
+long identifier = 123;
+int numberOfConnectedEdges = atlas.nodeForIdentifier(identifier).connectedEdges().size();
+```
+or
+```
+long identifier = 123;
+int numberOfConnectedEdges = atlas.nodeForIdentifier(identifier).absoluteValence();
+```
+
+# Building an `Atlas` from an `.osm.pbf` file
+
+## Generate an Atlas with everything that is in an `.osm.pbf` file
+
+```
+final File pbfFile;
+final OsmPbfLoader loader = new OsmPbfLoader(pbfFile, MultiPolygon.MAXIMUM,
+    AtlasLoadingOption.withNoFilter());
+final Atlas atlas = loader.read();
+```
+
+## Generate an Atlas without boundaries and coastlines
+
+Usually OSM boundaries and coastlines do not work well with JTS and country-slicing with themselves. The default `osmPbfWayFilter` in the `AtlasLoadingOption` will filter those out:
+
+```
+final File pbfFile;
+final OsmPbfLoader loader = new OsmPbfLoader(pbfFile, MultiPolygon.MAXIMUM,
+    AtlasLoadingOption.createOptionWithAllEnabled());
+final Atlas atlas = loader.read();
+```
+
+## Generate an Atlas for a specific shard
+
+When specifying a shard for building an `Atlas` from an `.osm.pbf` file, the generator will do a soft cut along the borders of the shard:
+
+```
+final File pbfFile;
+final Rectangle shard;
+final OsmPbfLoader loader = new OsmPbfLoader(pbfFile, MultiPolygon.forPolygon(shard),
+    AtlasLoadingOption.createOptionWithAllEnabled());
+final Atlas atlas = loader.read();
+```
+
+## Way Sectioning
+
+OSM ways usually span multiple intersections in the case of roads. To make the road network a navigable network, the process of loading an `.osm.pbf` file runs "way sectioning". It will follow a set of rules to break ways at intersections, and create `Atlas` `Edge`s. For that, it pads the OSM feature identifiers with six digits starting from 1 to the number of sections. For example, way 123 would become `Edge`s 123000001, 123000002 and 123000003 if it has to be broken twice.
```

Comment:
Small addition: "If no sectioning takes place, the edge identifier would end in 000"